src/modules/10-common/featureFlags.ts

```diff
@@ -28,6 +28,7 @@ export enum FeatureFlag {
   FFM_1827 = 'FFM_1827',
   FFM_3959_FF_MFE_Environment_Detail = 'FFM_3959_FF_MFE_Environment_Detail',
   FFM_5939_MFE_TARGET_GROUPS_LISTING = 'FFM_5939_MFE_TARGET_GROUPS_LISTING',
+  FFM_5943_FF_MFE_Environment_Listing = 'FFM_5943_FF_MFE_Environment_Listing',
```

Comment:
The flag should be named after the epic, not the individual ticket
```suggestion
  FFM_5256_FF_MFE_Environment_Listing = 'FFM_5256_FF_MFE_Environment_Listing',
```

Reply:
oops, updated now